ministryofjustice/laa-apply-for-criminal-legal-aid
# [CRIMAPP-912] Rename passporting benefit attribute to benefit check result

## Description of change
Ahead of rehydrating the benefit check attributes, this pr renames `passporting_benefit` to `benefit_check_result` for readability

## Link to relevant ticket

## Notes for reviewer
The benefit check service has a method called `passporting_benefit?` which runs the benefit check. We also have our own methods like `has_passporting_benefit?` which refers to whether an applicant has a benefit or not. Not sure if one of these should be amended (perhaps the benefit check service method name) as we're using passporting benefit for different things 

## Screenshots of changes (if applicable)

### Before changes:

### After changes:

## How to manually test the feature
No visible changes but you can start an application with a benefit and run the benefit check and then confirm that the attribute `benefit_check_result` is populated for the applicant in the rails console
